src/vs/workbench/contrib/inlineChat/browser/inlineChatStrategies.ts

```diff
@@ -383,6 +434,21 @@ export class LivePreviewStrategy extends LiveStrategy {
 		scrollState.restore(this._editor);
 	}
 
+	override getWidgetPosition(initialRender: boolean, range: Range, response: SessionResponse): Position | undefined {
+		if (initialRender) {
+			const viewModel = this._editor._getViewModel();
+			assertType(viewModel);
+			this._initialPosition = viewModel.getPrimaryCursorState().viewState.position;
+			return this._initialPosition;
+		} else {
+			if (range && response instanceof EditResponse) {
```

Comment:
Also better to use `_session.lastTextModelChanges` instead of the last edit response